### PR TITLE
Add typos for "general", "generate", "generic"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -19452,78 +19460,161 @@ gayity->gaiety
...
 geneation->generation
+geneations->generations
+geneator->generator
+geneators->generators
 geneic->generic, genetic,
 geneological->genealogical
 geneologies->genealogies
 geneology->genealogy
 generaates->generates
+generae->generate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:19529]: Perhaps `genera`, the plural of `genus`?
```
generae->generate, genera,
```
But then it's probably uncommon. Thoughts?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -19452,78 +19460,161 @@ gayity->gaiety
...
+genereally->generally
 genereate->generate
 genereated->generated
 genereates->generates
 genereating->generating
+genereation->generation
+genereations->generations
+genereator->generator
+genereators->generators
 genered->generated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:19573]: Just a suggestion, I'm letting you decide:
```
genered->generated, gendered,
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:19529]: I couldn't find an example of `generae->genera`, so I won't include this.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt]: I found a few examples of `genered->gendered` and `genered->gender`, so I added both.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**